### ISSUE TITLE
set-cookie-parser doesn't provide an export named default

### ISSUE THREAD

**Issue by selemondev** [Hebilicious/authjs-nuxt#26]
### Environment

```js
{
  "name": "nuxt-auth-app",
  "private": true,
  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev",
    "generate": "nuxt generate",
    "preview": "nuxt preview",
    "postinstall": "nuxt prepare"
  },
  "devDependencies": {
    "@nuxt/devtools": "latest",
    "@nuxtjs/color-mode": "^3.3.0",
    "@nuxtjs/google-fonts": "^3.0.1",
    "@nuxtjs/tailwindcss": "^6.8.0",
    "@pinia/nuxt": "^0.4.11",
    "@types/bcryptjs": "^2.4.2",
    "@types/node": "20.4.2",
    "nuxt": "3.6.3",
    "nuxt-icon": "^0.4.2",
    "typescript": "^5.1.6",
    "vue-tsc": "1.8.5"
  },
  "dependencies": {
    "@auth/core": "^0.9.0",
    "@hebilicious/authjs-nuxt": "0.1.4",
    "@vuelidate/core": "^2.0.3",
    "@vuelidate/validators": "^2.0.3",
    "bcryptjs": "^2.4.3",
    "joi": "^17.9.2",
    "mongoose": "7.3.4",
    "pinia": "^2.1.4",
    "vue-sonner": "^0.3.3"
  }
}
```

### Reproduction

https://codesandbox.io/p/sandbox/github/nuxt/starter/tree/v3

### Describe the bug

Upon completing the installation and configuration process and subsequently initiating my development server, I encountered an error message when inspecting my console: `utils.mjs:2 Uncaught SyntaxError: The requested module '/_nuxt/node_modules/set-cookie-parser/lib/set-cookie.js?v=4e1359b9' does not provide an export named 'default' (at utils.mjs:2:8)`

### Logs

```sh
`utils.mjs:2 Uncaught SyntaxError: The requested module '/_nuxt/node_modules/set-cookie-parser/lib/set-cookie.js?v=4e1359b9' does not provide an export named 'default' (at utils.mjs:2:8)`
```

**Comment by Hebilicious** [comment]
Please downgrade to 0.1.3 while I work on a fix

**Comment by selemondev** [comment]
OK

**Comment by Hebilicious** [comment]
@selemondev this should be fixed with 0.1.5

**Comment by selemondev** [comment]
Thank you.
